wikipedia-sentences-query-while-indexing/README.md

```diff
@@ -0,0 +1,152 @@
+
+You can check the `flows` and `pods` directories for the changes to the files.
+_____
+
+## 🐍 Build the app with Python
+
+These instructions explain how to run the example yourself and deploy it with Python. 
+
+### 🗝️ Requirements
+
+1. Have a working Python 3.7 environment.
+1. We recommend creating a [new Python virtual environment](https://docs.python.org/3/tutorial/venv.html) to have a clean installation of Jina and prevent dependency conflicts.   
+1. Have at least 2 GB of free space on your hard drive.
+
+### Running the example
+
+### 👾 Step 1. Clone the repo and install Jina
+
+Begin by cloning the repo so you can get the required files and datasets. (If you already have the examples repository on your machine make sure to fetch the most recent version)
+
+```sh
+git clone https://github.com/jina-ai/examples
+````
+
+And enter the correct folder:
+
+```sh
+cd examples/wikipedia-sentences-query-while-indexing
+```
+
+In your terminal, you should now be located in you the *wikipedia-sentences-query-while-indexing* folder. Let's install Jina and the other required Python libraries. For further information on installing Jina check out [our documentation](https://docs.jina.ai/chapters/core/setup/).
+
+```sh
+pip install -r requirements.txt
+```
+
+In order to run the example you will need to do the following:
+
+### 📥 Step 2. Download your data to search (Optional)
+
+The example repo includes a small subset of the Wikipedia dataset, for quick testing. You can just use that. 
+
+If you want to use the entire dataset, run `bash get_data.sh` and then modify the `DATA_FILE` constant (in `app.py`) to point to that file.
+
+### 🏃 Step 3. Running the Flows
+
+1. In one terminal session, run `jinad`.
+
+    `JinaD` is our platform for running Jina services (Flows, Pods) remotely, wherever you want to run them. In this example, we use `JinaD` to serve the two Flows (Index and Query). You can read more about it [in the docs](https://docs.jina.ai/chapters/remote/jinad).
+
+2. In another terminal, run `python app.py -t flows`
+
+    This will create the two Flows, and then repeatedly do the following, every 20 seconds:
+
+    1. index 5 Documents
+    2. send a `DUMP` request to the DBMS (Index) Flow to dump its data to a specific location
+    3. send a `ROLLING_UPDATE` request to the Query Flow to take down its Indexers and start them again, with the new data located at the respective path
+    
+    **Notice** the logs of the operations.
+
+    **Warning**: the data file is limited to 200 documents. Once that is exhausted, the process will terminate. If you want to use the entire dataset, run `bash get_data.sh` and then modify the `DATA_FILE` constant to point to that file.
+
+### 🔎 Step 4: Query your data
+
+Finally, in another terminal, run `python app.py -t client`
```

Comment:
in a third terminal 

- This clarify to the user how many terminals they should have open.